README.md

```diff
@@ -1 +1,209 @@
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/Itswali/React-Group.git .
+```
+
+
+### INSTALL
+
+Before installing the app, make sure to install node.js in your machine. In order to check if node is installed run the following command:
+
+```sh
+  node -v
+```
+
+Install this project with:
+
+```sh
+  npm install
+```
+
+### Usage
+
+To run the project in a development server, execute the following command:
+
+```sh
+  npm start
+```
+
+### Testing
+
+To run the tests in a development server, execute the following command:
+
+```sh
+  npm test
+```
+
+To to build for the production, execute the following command:
+
+```sh
+  npm run build
+```
+
+### Deployment
+
+You can deploy this project using [Github Pages](https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-github-pages-site)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Wali-Muhammad**
+
+- GitHub: [@Itswali](https://github.com/Itswali)
+- Twitter: [@Wali-Muhammad](https://twitter.com/WaliMuh94818599)
+- LinkedIn: [@Wali Muhammad](https://www.linkedin.com/in/wali-muhammad-666040244/)
+
+👤 **Hein Thant**
+
+- GitHub: [@githubhandle](https://github.com/indiecodermm)
+- Twitter: [@twitterhandle](https://twitter.com/hthant_oo)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/hthantoo)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **Add new missiles**
+- [ ] **Feature checking all the reserved rockets and missions**
+- [ ] **Cancelling reservation on my profile page**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
+
+If you like this project then give it a star ⭐️ 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- FAQ (optional) -->
+<!--
+## ❓ FAQ (OPTIONAL) <a name="faq"></a> -->
+<!--
+
+> Add at least 2 questions new developers would ask when they decide to use your project.
+
+- **How to install all the packages used in the app**
+
+  - Just type the command npm install It will automatically install all the dependencies
+
+- **How to Open on browser**
+
+  - npm start coommand will do the trick for you.
+  -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- Hey, since you commented out this section, please, remove them from your readme file to make it clean, also, remove the link (back to top) since the section is not visible in the readme file. Let's do this to make our readme more professional.